internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -12,135 +12,85 @@ import (
-		projectName = acc.RandomProjectName() // No ProjectIDExecution to avoid cross-region limits because multi-region
-		clusterName = acc.RandomClusterName()
-		config      = configReplicaSetMultiCloud(orgID, projectName, clusterName)
-	)
-
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { mig.PreCheckBasic(t) },
-		CheckDestroy: acc.CheckDestroyCluster,
-		Steps: []resource.TestStep{
-			{
-				ExternalProviders: mig.ExternalProviders(),
-				Config:            config,
-				Check:             checkReplicaSetMultiCloud(clusterName, 3, false),
-			},
-			mig.TestStepCheckEmptyPlan(config),
-			{
-				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-				Config:                   config,
-				Check:                    checkReplicaSetMultiCloud(clusterName, 3, true), // external_id will be present in latest version
-			},
-		},
-	})
+	// once 1.18.0 is released we can adjust this to always check new attributes - CLOUDP-266096
+	testCase := replicaSetMultiCloudTestCase(t, false)
+	mig.CreateAndRunTest(t, &testCase)
 }
 
 func TestMigAdvancedCluster_singleShardedMultiCloud(t *testing.T) {
-	var (
-		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName = acc.RandomProjectName() // No ProjectIDExecution to avoid cross-region limits because multi-region
-		clusterName = acc.RandomClusterName()
-		config      = configShardedMultiCloud(orgID, projectName, clusterName, 1, "M30")
-	)
+	// once 1.18.0 is released we can adjust this to always check new attributes - CLOUDP-266096
+	testCase := singleShardedMultiCloudTestCase(t, false)
+	mig.CreateAndRunTest(t, &testCase)
+}
 
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { mig.PreCheckBasic(t) },
-		CheckDestroy: acc.CheckDestroyCluster,
-		Steps: []resource.TestStep{
-			{
-				ExternalProviders: mig.ExternalProviders(),
-				Config:            config,
-				Check:             checkShardedMultiCloud(clusterName, 1, "M30", false),
-			},
-			mig.TestStepCheckEmptyPlan(config),
-			{
-				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-				Config:                   config,
-				Check:                    checkShardedMultiCloud(clusterName, 1, "M30", true), // external_id will be present in latest version
-			},
-		},
-	})
+func TestMigAdvancedCluster_symmetricGeoShardedOldSchema(t *testing.T) {
+	// once 1.18.0 is released we can adjust this to always check new attributes - CLOUDP-266096
+	testCase := symmetricGeoShardedOldSchemaTestCase(t, false)
+	mig.CreateAndRunTest(t, &testCase)
+}
+
+func TestMigAdvancedCluster_asymmetricShardedNewSchema(t *testing.T) {
+	acc.SkipTestForCI(t) // latest release does not support ISS, to be adjusted in CLOUDP-266096
+	testCase := asymmetricShardedNewSchemaTestCase(t)
+	mig.CreateAndRunTest(t, &testCase)
 }
 
-func TestMigAdvancedCluster_singleShardPerZoneGeoSharded(t *testing.T) {
+func TestMigAdvancedCluster_replicaSetAWSProviderUpdateAfterVerisonUpgrade(t *testing.T) {
```

Comment:
```suggestion
func TestMigAdvancedCluster_replicaSetAWSUpdate(t *testing.T) {
```

Reply:
since these are migration tests (which are used for comparison between version upgrades maybe we can exclude that from the name)

Reply:
agree, keeps it more simple